Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -792,22 +792,26 @@ void MainWindow::message(QString message, QString colorName, QString font) {
     message.remove(message.length()-1, 1);
   }
   statusBar()->showMessage(message, 5000);
+  QTimer::singleShot(5000, this, SLOT(resetStatusColor()));
```

Comment:
As `resetStatusColor()` is used only once, here, that is a good opportunity to use a lambda expression, instead of a new slot: since Qt-5.4, `QWidget::singleShot` can be connected to lambda expressions.